tests/src/test/scala/com/nvidia/spark/rapids/HostCoalesceBatchesSuite.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids
+
+import org.scalatest.FunSuite
+
+class HostCoalesceBatchesSuite extends FunSuite with Arm {
+  test("close handling of host batches") {
+
+    // Host columnar batches are closed by the producer, not the consumer.
+    val iterchain = AutoCloseColumnBatchIterator
```

Comment:
was this committed by mistake (perhaps missing a diff)?

Reply:
Oops, cleaned this up.